VAMobile/src/screens/HealthScreen/Appointments/PastAppointments/PastAppointments.tsx

```diff
@@ -171,6 +165,15 @@ function PastAppointments({
   const pickerOptions = getPickerOptions()
   const [datePickerOption, setDatePickerOption] = useState(pickerOptions[0])
 
+  useEffect(() => {
```

Comment:
Alternately, we can set up a different useEffect for just the datePickerOption and the data and set it into a state variable that this uses.